motioneye/mmalctl.py

```diff
@@ -39,8 +39,12 @@ def list_devices():
     except subprocess.CalledProcessError:  # not found
         return []
 
-    d = dict(p.split('=', 1) for p in support.split())
-    if d.get('detected') == d.get('supported') == '1':
+    # Temporary patch for camera detection on MMAL going wrong due to split failing
+    # The response from vcgencmd is "supported=1 detected=1, libcamera interfaces=0"
+    # split fails on the "libcamera interfaces" part originally. The comma was overlooked too.
+    # see https://github.com/ccrisan/motioneye/pull/2305#issuecomment-1058584622
+    d = dict(p.split('=', 1) for p in support.split(' ', 2))
+    if d.get('detected').strip(',') == d.get('supported').strip(',') == '1':
```

Comment:
This splitting looks overly complicated to me. What about simply checking whether the whole output starts with the string we expect?
```suggestion
if support.startswith('supported=1 detected=1'):
```

Reply:
Makes sense.
The splitting comes from how it was (with the splitting and double check), the simplified starts with is a lot better indeed.